Showcase/Showcase/Pages/Tokens/Color/NamedColor/NamedColor+OnContent.swift

```diff
@@ -0,0 +1,82 @@
+        case colorContentOnBackgroundStatusAccentMuted
+        case colorContentOnBackgroundStatusAccentMutedOnBackgroundEmphasized
+        case colorContentOnBackgroundStatusAccentedEmphasized
+
+        func token(from theme: OUDSTheme) -> ColorSemanticToken {
+            switch self {
+            case .colorContentOnBackgroundPrimary:
+                return theme.colorContentOnBackgroundPrimary
+//            case .colorContentOnBackgroundSecondary:
+//                return theme.colorContentOnBackgroundSecondary
+//            case .colorContentOnBackgroundTertiary:
+//                return theme.colorContentOnBackgroundTertiary
+            case .colorContentOnBackgroundStatusPositiveMuted:
+                return theme.colorContentOnBackgroundStatusPositiveMuted
+            case .colorContentOnBackgroundStatusPositiveMutedOnBackgroundEmphasized:
+                return theme.colorContentOnBackgroundStatusPositiveMutedOnBackgroundEmphasized
+            case .colorContentOnBackgroundStatusPositiveEmphasized:
+                return theme.colorContentOnBackgroundStatusPositiveEmphasized
+            case .colorContentOnBackgroundStatusInfoMuted:
+                return theme.colorContentOnBackgroundStatusInfoMuted
+            case .colorContentOnBackgroundStatusInfoMutedOnBackgroundEmphasized:
+                return theme.colorContentOnBackgroundStatusInfoMutedOnBackgroundEmphasized
+            case .colorContentOnBackgroundStatusInfoEmphasized:
+                return theme.colorContentOnBackgroundStatusInfoEmphasized
+            case .colorContentOnBackgroundStatusWarningMuted:
+                return theme.colorContentOnBackgroundStatusWarningMuted
+            case .colorContentOnBackgroundStatusWarningMutedOnBackgroundEmphasized:
+                return theme.colorContentOnBackgroundStatusWarningMutedOnBackgroundEmphasized
+            case .colorContentOnBackgroundStatusWarningEmphasized:
+                return theme.colorContentOnBackgroundStatusWarningEmphasized
+            case .colorContentOnBackgroundStatusNegativeMuted:
+                return theme.colorContentOnBackgroundStatusNegativeMuted
+            case .colorContentOnBackgroundStatusNegativeMutedOnBackgroundEmphasized:
+                return theme.colorContentOnBackgroundStatusNegativeMutedOnBackgroundEmphasized
+            case .colorContentOnBackgroundStatusNegativeEmphasized:
+                return theme.colorContentOnBackgroundStatusNegativeEmphasized
+            case .colorContentOnBackgroundStatusAccentMuted:
+                return theme.colorContentOnBackgroundStatusAccentMuted
+            case .colorContentOnBackgroundStatusAccentMutedOnBackgroundEmphasized:
+                return theme.colorContentOnBackgroundStatusAccentMutedOnBackgroundEmphasized
+            case .colorContentOnBackgroundStatusAccentedEmphasized:
+                return theme.colorContentOnBackgroundStatusAccentedEmphasized
+            }
+        }
+    }
+}
```

Comment:
Normalement tu dois avoir des warnings tels que :
- "Blanket Disable Command Violation: The disabled 'identifier_name' rule should be re-enabled before the end of the file (blanket_disable_command)"

Les warnings désactivés dans un fichier doivent être réactivés ensuite.

Il faut ajouter à la fin du fichier, après un saut de ligne, les lignes : 

```swift
// swiftlint:enable identifier_name
```